Dockerfile

```diff
@@ -67,12 +78,17 @@ RUN set -x && \
         tar -xz -C /opt/ && \
     mv /opt/hadoop-${HADOOP_VERSION} /opt/hadoop && \
     echo "Downloading Spark" && \
-    wget -qO - http://apache.stu.edu.tw/spark/spark-${SPARK_VERSION}/spark-${SPARK_VERSION}-bin-hadoop${HADOOP_MAJOR_MINOR_VERSION}.tgz |\
+    wget -qO - http://archive.apache.org/dist/spark/spark-${SPARK_16_VERSION}/spark-${SPARK_16_VERSION}-bin-hadoop${HADOOP_MAJOR_MINOR_VERSION}.tgz |\
```

Comment:
To reduce the size of the image, maybe we should conditionally install 1.6 or 2.0 only. Otherwise the image is quite fat.